next.config.js

```diff
@@ -14,7 +14,7 @@ const nextConfig = {
     return config;
   },
   images: {
-    domains: ["dummyimage.com"],
+    domains: ["images.unsplash.com", "unsplash.com"],
```

Comment:
Habt ihr das mal selbst getestet? Ich sehe nämlich nicht, dass es funktioniert. Schaut euch gerne mal die docs zur Image Komponente von next an. Hier ein Auszug:
![Screenshot 2023-11-24 at 11 27 10](https://github.com/feivlinus/fork-yeah/assets/111574974/95315108-925b-48e3-943a-45bbb81f0f19)

Reply:
Hallo Roland. 
Ja wir haben es vorher getestet.
Man muss eine Image URL verwenden. Rechtsklick auf das Bild, auf Grafikadresse kopieren klicken und in das Formularfeld rein kopieren.

Du hast du einen Link geschickt zu einer Seite von einem Bild. Wir haben auch dies probiert und sämtliche Variationen an URLs ausprobiert. Man kann keine komplette Seite der Image Komponente als src geben.
![Uploading proof.png…]()

Reply:
Server log:
`The requested resource isn't a valid image for https://unsplash.com/photos/a-white-plate-topped-with-french-toast-and-raspberries-ukvx50g_opc received text/html; charset=utf-8`

Reply:
Ah! Okay. Ich habs getestet und so läufts. Super!